style/client.css

```diff
@@ -197,19 +197,44 @@ select {
 }
 */
 .dark .button,
+.dark .folderButton,
 .dark .tabbar a.button,
+.dark .popupmenu button.folderButton,
 .dark .popupmenu button.button {
 	background: #484848;
 	box-shadow: inset 0 3px 4px rgba(255, 255, 255, 0.25);
 	border-color: #A9A9A9;
 	text-shadow: none;
 	color: #F9F9F9;
 }
+
+.dark .popupmenu button.folderButtonOpen {
```

Comment:
this can be added to line 203 and then you can just make this:

```
background: linear-gradient(to bottom, #2a2a2a, #313131);
```

because I believe thats the only difference?